lib/core/di/services_module.dart

```diff
@@ -15,7 +15,8 @@ Future<void> _registerServicesModule() async {
 
   _registerLazySingleton(() => CryptoAuthService());
 
-  _registerLazySingleton(() => EOSClient(baseUrl: Endpoints.baseUrl, version: 'v1'));
+  _registerLazySingleton(
+      () => EOSClient(baseUrl: GetIt.I<RemoteConfigService>().pushTransactionNodeUrl(), version: 'v1'));
```

Comment:
Look around this class how other injects are done., follow that pattern

_getIt<RemoteConfigService>()

Reply:
are you going to make `EOSClient` a regular class and not an injected singleton. Doesnt have to be in this PR, just checking we are doing that